### ISSUE TITLE
upstream?

### ISSUE THREAD

**Issue by Fuseteam** [kz6fittycent/bashtop#1]
is there a change the original author will pick this up?

**Comment by kz6fittycent** [comment]
I'm not sure, why do you ask?

**Comment by Fuseteam** [comment]
would be nice if he supports/endorses it too. would make it easier to keep up to date perhaps? i would expect there to be some benefit if you and the upstream dev collaborate more closely.

**Comment by kz6fittycent** [comment]
As far as keeping it up to date, the yaml points to his repo and any time he makes a commit, it immediately builds a new snap. This is then published (automatically) to the `--edge` channel. 

That said, I reached out to Aristocratos and let him know that I've got both `bashtop` and `bpytop` snapped and if he wants to take them over, we can transfer them both to him. 

It's been my experience that most devs are happy to let someone else worry about it, so it's become a low priority for me to bother with it. 

Thanks for using the snap and stay tuned to see what Aristocratos wants to do. 

I'll close this issue as it really depends on what he has to say.
